Modules/Registration/Common/test/itkMeanSquaresImageMetricTest.cxx

```diff
@@ -295,6 +298,110 @@ itkMeanSquaresImageMetricTest(int, char *[])
   //-------------------------------------------------------
   metric->Print(std::cout);
 
+
+   //-------------------------------------------------------
```

Comment:
I know these empty comment lines are being used already in the test, but please remove them from the new code.